controller/src/main/java/io/pravega/controller/store/stream/Stream.java

```diff
@@ -134,19 +135,36 @@
     CompletableFuture<List<Integer>> getActiveSegments(final long timestamp);
 
     /**
+     * Returns the currently active stream epoch.
+     * @param epoch epoch number
+     * @return currently active stream epoch.
+     */
+    CompletableFuture<List<Integer>> getActiveSegments(int epoch);
+
+    /**
      * Called to start metadata updates to stream store wrt new scale event.
      *
      * @param newRanges      key ranges of new segments to be created
      * @param scaleTimestamp scaling timestamp
+     * @param runOnlyIfStarted run only if scale is started
```

Comment:
This parameter needs explanation. Please add description for  this param, as it is an important design choice.